tests/arp/test_stress_arp.py

```diff
@@ -30,6 +31,34 @@
 }
 
 
+@pytest.fixture(autouse=True)
+def arp_cache_fdb_cleanup(duthost):
+    try:
```

Comment:
What is the significance of this auto used fixture? As in your description, "Sometimes arp cache cleanup will fail if it is done too often", which means this fixture could also fail.

Reply:
The functions `clear_dut_arp_cache()` and `fdb_cleanup()` may fail - which is why there are in a try-except in the fixture.

This fixture exists in addition to the arp and fdb clear ran in the test function to ensure arp cache and fdb is clean before the test function is ran. Currently the clean up is only after the test is done.

Reply:
If it fails with `file not found`, then it means it is fine as the file is a cached file on the sonic-mgmt docker. No file means there is nothing or cache and/or the file has been recently deleted - meaning recently cleared/flushed.

If it does not fail with `file not found`, then the test should fail as it means there is something else wrong that needs to be triaged.

Reply:
What I'm considering is that, also we have this fixture, the entries still can not be cleaned up due to some reasons.

Reply:
I could catch the other failures and log it instead of throwing another - is that preferred over fast failing?

Reply:
But in this way, we can not clear the entries, right?